README.md

```diff
@@ -6,7 +6,11 @@
 
 A collection of lints to catch common mistakes and improve your [Rust](https://github.com/rust-lang/rust) code.
 
-[There are 352 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
+<<<<<<< HEAD
+[There are 354 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
+=======
+[There are 354 lints included in this crate!](https://rust-lang.github.io/rust-clippy/master/index.html)
+>>>>>>> add excessive bools lints
```

Comment:
Rebase gone wrong. LGTM otherwise.

Reply:
Ooh, I thought I can just `cargo dev update_lints` and it will erase those from everywhere. I'll maybe do a pr to make it do that then

Reply:
I think it couldn't detect this line in this case.

Reply:
It did detect the line, that's why they are both on 354